src/tokens.md

```diff
@@ -283,7 +283,7 @@ following forms:
 >
 > RAW_BYTE_STRING_CONTENT :\
 > &nbsp;&nbsp; &nbsp;&nbsp; `"` ASCII<sup>* (non-greedy)</sup> `"`\
-> &nbsp;&nbsp; | `#` RAW_STRING_CONTENT `#`
+> &nbsp;&nbsp; | `#` RAW_BYTE_STRING_CONTENT `#`
```

Comment:
`RAW_BYTE_STRING_CONTENT` should be a recursive definition (like `RAW_STRING_CONTENT`)